modules/admin_manual/pages/configuration/files/external_storage/configuration.adoc

```diff
@@ -61,6 +60,9 @@ users and groups in the *Available for* field.
 
 image:configuration/files/external_storage/applicable.png[User and groups selector]
 
+IMPORTANT: Adding a storage for users or groups, you don't have access rights your self will result in an error notification and icon.
```

Comment:
Proposal:
`Adding a storage for users or groups, you don't have access rights your self will result in an error notification and icon.` -->
`Adding a storage for users or groups you don't have access rights to, a error notification will be shown and a red square icon appears on the mount.`